django_multitenant/mixins.py

```diff
@@ -52,7 +52,6 @@ def bulk_create(self, objs, **kwargs):
 
 class TenantModelMixin:
     # Abstract model which all the models related to tenant inherit.
-    tenant_id = ""
```

Comment:
Interesting that this line of code goes all the way back to [when the mixin was created](https://github.com/citusdata/django-multitenant/commit/5085adb47d486d604269c4c235ae1a283b26dafe#diff-78923b7bd47cd6a31bcbdd597583045a38eee5a22f315e5cf7ed6a908ad5e12aR92).

Reply:
and it is unnecessary. I found lots of unused codes like this one. These codes make the project hard to maintain. I deleted/changed lots of codes like that and will continue If I will find in the future